### PR TITLE
Add add functionality and column list for ZGW

### PR DESCRIPTION
**Pull request by teodor-ritense**
[[FE] Document list columns management](https://ritense.tpondemand.com/entity/107913-fe-document-list-columns-management)

PARTIAL FUNCTIONALITY:
- All edit and sorting stuff is pending on another story

PLEASE CHECK TRANSLATIONS

### CODE REVIEW

File: projects/valtimo/config/assets/core/nl.json [hunk 1]
```diff
@@ -628,6 +628,17 @@
       }
     }
   },
+  "zgw": {
+    "columns": {
+      "column": "Kolom",
+      "add": "Kolom toevoegen",
+      "edit": "Kolom bewerken",
+      "deleteText": "Weet je zeker dat je dit kolom wilt verwijderen?"
```
Comment (peetritense) [inline projects/valtimo/config/assets/core/nl.json]:
```suggestion
      "deleteText": "Weet je zeker dat je deze kolom wilt verwijderen?"
```

File: projects/valtimo/config/assets/core/nl.json [hunk 2]
```diff
@@ -628,6 +628,17 @@
...
+    },
+    "tabs": {
+      "documentColumns": "Documentkolommen"
```
Comment (peetritense) [inline projects/valtimo/config/assets/core/nl.json]:
```suggestion
      "documentColumns": "Document kolommen"
```

File: projects/valtimo/config/assets/core/nl.json
Comment (peetritense) [inline projects/valtimo/config/assets/core/nl.json:1]: I'm missing the "defaultSort": "Standaard sortering", on line 825 (?)

File: projects/valtimo/config/assets/core/de.json
Comment (peetritense) [inline projects/valtimo/config/assets/core/de.json:1]: I'm missing the "defaultSort": "Standaard sortering", on line 825 (?)

**Review by peetritense — CHANGES_REQUESTED** [review]

File: projects/valtimo/config/assets/core/nl.json
Comment (teodor-ritense) [reply projects/valtimo/config/assets/core/nl.json:1]: Will add it when the defaulSorting is available functionality wise

**Review by peetritense — APPROVED** [review]

**Review by sofiaIvarsRitense — APPROVED** [review]

**Pull request merged**